truss/build.py

```diff
@@ -122,6 +205,30 @@ def from_directory(truss_directory: str) -> TrussHandle:
     return TrussHandle(Path(truss_directory))
 
 
+def mk_truss(
+    obj: Any,
+    target_directory: str = None,
+    data_files: List[str] = None,
+    requirements_file: str = None,
+    bundled_packages: List[str] = None,
+) -> TrussHandle:
+    # Some model objects can are callable (like Keras models)
+    # so we first attempt to make Truss via a model object
+    try:
+        return mk_truss_from_model(
+            obj, target_directory, data_files, requirements_file, bundled_packages
+        )
+    except FrameworkNotSupportedError:
```

Comment:
minor: To avoid using try for control flow, we can wrap `mk_truss_from_model` into a separate function that returns None on failure and use the null check for control flow here.

```def mk_truss_from_model_opt(...):
	# returns None if framework not supported, otherwise the truss
```